src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -0,0 +1,137 @@
+
+In this version of the connector:
+- the country code will be set to the fixed value `FR`,
+- the city code will be set to the fixed value `XXXXX`,
+- the technical code of the stop will contain the value of `stop_id`, with a replacement of `:` by `_` to avoid conflict between separators,
+- the code of the provider will be set to the value of `StopProviderCode` param (is specified) or  `LOC` otherwise.
+
+
+**(3) definition of SiteRef**
+TBD
+
+**(2) definition of Location**
+The GTFS stop_lon and stop_lat are specified in WGS84. The coordinates are converted to EPSG:2154 (Lambert 93).  
+Example of Netex declaration:
+><gml:pos srsName="EPSG:2154">662233.0 6861519.0</gml:pos>
+
+**(3) definition of the TransportMode**
+As a stop_point can be associated to several physical_modes, all the physical_modes need to be mapped to the Netex list.
+If more than one Netex mode is associated, the most frequent one is used and a warning is emitted.
+
+physical_mode_id | TransportMode in Netex
+--- | --- 
+Air | air
+Boat | water
+Bus | bus
+BusRapidTransit | bus
+Coach | coach
+Ferry | water
+Funicular | funicular
+LocalTrain | rail
+LongDistanceTrain | rail
+Metro | metro
+RapidTransit | rail
+RailShuttle | rail
+Shuttle | bus
+SuspendedCableCar | cableway
+Taxi | _ignored_
+Train | train
+Tramway | tram
+
+#### AccessibilityAssessment
+
+If the stop_point is associated to an equipment, a node `AccessibilityAssessment` is created and it's content is as follow:
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+AccessibilityAssessment/@id | stops.txt | equipment_id | 
+AccessibilityAssessment/MobilityImpairedAccess | | see (1) below
+AccessibilityAssessment/ limitations/AccessibilityLimitation/ WheelchairAccess | equipments.txt | wheelchair_boarding | see (2) below
+AccessibilityAssessment/ limitations/AccessibilityLimitation/ AudibleSignsAvailable | equipments.txt | audible_announcement | see (2) below
+AccessibilityAssessment/ limitations/AccessibilityLimitation/ VisualSignsAvailable | equipments.txt | visual_announcement | see (2) below
+
+**(1) definition of MobilityImpairedAccess**
+As stated in `NF_Profil NeTEx éléments communs(F) - v2.1.pdf` in chapter 5.10:
+`AccessibilityAssessment` is optionnal, but if it's present, `MobilityImpairedAccess` is mandatory. Therefore, `MobilityImpairedAccess` sould be set to:
+- `true` if all `AccessibilityLimitation` are set to `true`
+- `false` if all `AccessibilityLimitation` are set to `false`
+- `partial` if some `AccessibilityLimitation` are set to `true`
+- `unknow` in other cases
+
+**(2) value of limitations**
+
+NTFS accessibility value | Netex accessibility value
+--- | ---
+0 or undefined | `undefined` 
+1 | `true`
+2 | `false`
+
+### monomodalStopPlace and multimodalStopPlace
+warnings: 
+monomodalStopPlace: tous les arrêts physiques ont le meme nom, sinon il faut en créer 2
+multimodalStopPlace: monter le mode de plus haut niveau
```

Comment:
I don't understand this language

Reply:
yes, it was personal notes for later :p switching to `en`